cogs/send_introduction_reminders.py

```diff
@@ -274,7 +277,8 @@ async def opt_out_introduction_reminders_button_callback(self, button: discord.B
                     )
                 except ValidationError as create_introduction_reminder_opt_out_member_error:
                     error_is_already_exists: bool = (
-                        "hashed_member_id" in create_introduction_reminder_opt_out_member_error.message_dict  # noqa: E501
+                        "hashed_member_id"
+                        in create_introduction_reminder_opt_out_member_error.message_dict
```

Comment:
Comparison statements should not be split between lines. See above comment for further reasoning.